lib/inets/src/http_client/httpc_response.erl

```diff
@@ -194,7 +194,7 @@ parse_status_code(<<?LF>>, StatusCodeStr,
 		      MaxHeaderSize, Result, true);
 
 parse_status_code(<<?CR, ?LF, Rest/binary>>, StatusCodeStr, 
-		  MaxHeaderSize, Result, true) ->
+		  MaxHeaderSize, Result, true) when is_list(StatusCodeStr) ->
```

Comment:
Do we really need this guard? We usually trust internal code and do not guard it!